portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,45 +14,70 @@
 
 package com.google.sps.servlets;
 
-
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.gson.Gson;
 import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-
  class Comment{
     private String name;
     private String lastname;
     private String comment;
 
+    Comment(String n, String ln, String c){
```

Comment:
Better variable names, since these don't convey much to the reader.

Reply:
I have updated the names, let me know if you have any more suggestions.